app/mailers/admin_mailer.rb

```diff
@@ -49,7 +49,19 @@ def set_password_notification(admin, token)
 
     mail(
       to: @admin.email,
-      subject: t(".subject", app_name: ArchiveConfig.APP_SHORT_NAME)
+      subject: default_i18n_subject(app_name: ArchiveConfig.APP_SHORT_NAME) 
+    )
+  end
+
+  # Sends an email to the admin who lost their 2FA device when a sysadmin runs
+  # the script in script/send_backup_codes_to_admin.rb
+  def totp_2fa_backup_codes(admin, codes)
```

Comment:
Please add a rspec test for this mail, similar to the `set_password_notification` test